frontend/src/components/common/GenericModal.tsx

```diff
@@ -49,7 +49,7 @@ interface ModalProps {
   /** Optional title to display - no default. */
   title?: string;
   /** Optional message to display - no default. */
-  message?: string;
+  message?: string | React.ReactNode;
```

Comment:
Interesting

Reply:
Yeah, this is in keeping with what some of the react-bootstrap components do. For example, tab titles can be either strings or nodes. That allows you to embed things like icons in the title. This makes sense for cases where passing the component as a child doesn't really make sense.